ReactWindows/ReactNative.Shared/Views/Control/ReactControlManager.cs

```diff
@@ -95,6 +95,24 @@ public override string  Name
             }
         }
 
+        /// <inheritdoc />
+        public override IReadOnlyDictionary<string, object> ExportedCustomDirectEventTypeConstants
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    {
+                        "topAccessibilityTap",
+                        new Dictionary<string, object>
+                        {
+                            { "registrationName", "onAccessibilityTap" },
+                        }
+                    },
+                };
+            }
+        }
+
```

Comment:
You don't need this, you already defined the event in UIManagerModule.Constants.cs

Reply:
I see. I just followed existing code. For example ReactFlipViewManager does this although the event it defines there is already in UIManagerModule.Constants.cs. Should we change all the other places not to duplicate declaration too then?

---
In reply to: [158572561](https://github.com/reseul/react-native-windows/pull/1#discussion_r158572561) [](ancestors = 158572561)

Reply:
removed it from here in order for this to be global since it's not specific to ReactControl.

---
In reply to: [159557719](https://github.com/reseul/react-native-windows/pull/1#discussion_r159557719) [](ancestors = 159557719,158572561)